packages/cli/src/Generate.ts

```diff
@@ -246,8 +255,18 @@ ${chalk.dim('```')}
 
 You will need a Prisma Data Proxy connection string. See documentation: ${link('https://pris.ly/d/data-proxy')}
 `
-            : ''
-        }${breakingChangesStr}${versionsWarning}`
+              : ''
+          }${breakingChangesStr}${versionsWarning}`
+        } else {
+          hint = `You can now start using Prisma Client in your code. Reference: ${link('https://pris.ly/d/client')}
+${chalk.dim('```')}
+${highlightTS(`\
+import { PrismaClient } from '${importPath}/deno/edge.ts'`)}
+${chalk.dim('```')}
+
+You will need a Prisma Data Proxy connection string. See documentation: ${link('https://pris.ly/d/data-proxy')}
+${breakingChangesStr}${versionsWarning}`
```

Comment:
I don't like that deno previw is in the `else`. Can we rewrite this that this also reacts to `isDenoPreview` @millsp?